TODO.md

```diff
@@ -6,28 +6,29 @@ You can find completed versions at [the bottom of the list](#Completed).
 
 - Create users file if it does not exist
 
+## Bugs
+- [x] Choosing not to exit program on first attempt 
 
 ## 0.0.2: The Everything Update
 
-- [ ] Add CI/CD deployments for code quality & error checking. @TechnoShip123
+- [x] Add CI/CD deployments for code quality & error checking. @TechnoShip123
 - [ ] Proper `master/` and `feature/xyz` branch structuring (`dev/` likely unnecessary, maybe if the project gains more traction). @TechnoShip123
 Add [badges for build status](https://shields.io/category/build) when done.
```

Comment:
Move the second part of this task to its own checkbox and check it off as completed.
```diff
- Add [badges for build status](https://shields.io/category/build) when done.
+ - [x] Add [badges for build status](https://shields.io/category/build) when done.
```

Reply:
Fixed in [b5cdf40](https://github.com/TechnoShip123/numbersense/pull/6/commits/b5cdf40f493136bfca41b2ee092617f19096706e)